aiaccel/config/config.py

```diff
@@ -168,13 +179,17 @@ def pathlib2str_config(config: ListConfig | DictConfig) -> ListConfig | DictConf
     def _inner_fn(config: ListConfig | DictConfig) -> ListConfig | DictConfig:
         if isinstance(config, ListConfig):
             for ii in range(len(config)):
-                config[ii] = _inner_fn(config[ii])
+                node = config._get_node(ii)
+                if isinstance(node, ocNode) and node is not None and not node._is_interpolation():
+                    config[ii] = _inner_fn(config[ii])
         elif isinstance(config, DictConfig):
             for k, v in config.items():
-                if isinstance(v, ListConfig | DictConfig):
-                    config[k] = _inner_fn(v)
-                elif isinstance(v, Path):
-                    config[k] = str(v)
+                node = config._get_node(k)
+                if isinstance(node, ocNode) and node is not None and not node._is_interpolation():
```

Comment:
こちらこう書けますかね？ 
```suggestion
                if isinstance(node := config._get_node(k), ocNode) and not node._is_interpolation():
```
ocNodeはNodeで良いように思いました🙇‍♂️